submissions/kolawoleOgunfowokan/HTML/js/script.js

```diff
@@ -0,0 +1,58 @@
+let productImage = document.getElementsByClassName("images");
+let prodTitle = document.getElementsByClassName("product-title");
+let prodPrice = document.getElementsByClassName("product-price");
+let display = document.getElementById("product-box");
+display.addEventListener("click", displayProduct);
+
+
+
+let productArray = [
+     {image: "../images/HP250.jpg",
```

Comment:
Please indent this properly. Objects should look similar to:
```
{
    image: "../images/eutetic-freezer.jpg",
    title: "Freezer",
    price: "N195000"
}
```

Reply:
I have fixed the indenting issues too.